proposals/p3848.md

```diff
@@ -0,0 +1,576 @@
+# Lambdas
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+[Pull request](https://github.com/carbon-language/carbon-lang/pull/3848)
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Abstract](#abstract)
+-   [Syntax Overview](#syntax-overview)
+-   [Introducer](#introducer)
+-   [Positional Parameters](#positional-parameters)
+    -   [Positional Parameter Restrictions](#positional-parameter-restrictions)
+-   [Function Captures](#function-captures)
+    -   [Default Capture Mode](#default-capture-mode)
+    -   [Capture Modes](#capture-modes)
+-   [Function Fields](#function-fields)
+-   [Copy Semantics](#copy-semantics)
+-   [Self and Recursion](#self-and-recursion)
+-   [Rationale](#rationale)
+
+<!-- tocstop -->
+
+## Abstract
+
+To support migration from C++ to Carbon, there must be valid syntax to capture
+the behavior of C++ lambdas. They are defined at their point of use and are
+often anonymous, meaning replacing them solely with function declarations will
+create an ergonomic burden compounded by the need for the migration tool to
+select a name. This proposal proposes a path forward to add lambdas to Carbon
+and augment function declarations accordingly.
```

Comment:
I think of an abstract as a kind of summary of the document, but most of the content here is unique to the abstract, as far as I can tell. I'd reccommend moving most of this to the Rationale section, and focus the Abstract more on the key points of what's being proposed.

Reply:
Done! Should I move Rationale to the top or leave it at the bottom?